solutions/1630/01.py

```
@@ -1,26 +1,23 @@
-from typing import List
-
 class Solution:
     def checkArithmeticSubarrays(self, nums: List[int], l: List[int], r: List[int]) -> List[bool]:
```

Ruff (0.14.8)

Undefined name `List`

(F821)